src/Mod/TechDraw/Gui/QGIViewDimension.cpp

```diff
@@ -266,8 +270,8 @@ void QGIDatumLabel::setPosFromCenter(const double &xCenter, const double &yCente
     }
     double tolRight = unitRight + width;
 
-    m_tolTextOver->justifyRightAt(tolRight, middle, false);
-    m_tolTextUnder->justifyRightAt(tolRight, middle + underBox.height(), false);
+    m_tolTextOver->justifyRightAt(tolRight, middle - overBox.height() + underBox.height()/4, false);
```

Comment:
A comment about why the division by 4 is here might aid future developers.

Reply:
Many thanks for your reviews. What you uncover are indeed problems. For now it might be clear what the 4 is about but next year it will be hard to understand why we choose 4 and e.g. not 3 or 5.

Reply:
I agree.  I made a new commit revealing the general calculation logic, without just delivering magic numbers into the code.  Except the 2\*y_adj, which is a font thing, but at least it is commented now in the code.

Reply:
Many thanks. from my side it can go in.